### ISSUE TITLE
- (NSURL *)imageURLWithDimension:(NSUInteger)dimension; Call Crash 2.3.0 iOS

### ISSUE THREAD

**Issue by sauvikatinnofied** [googlesamples/google-services#77]
I have just upgraded to Sign In SDK to 2.3.0 from 2.2.0. I need to get user's profile picture URL as string from `GIDGoogleUser` 's  `profileData`. Earlier it was working fine, but now its crashing the application. 

Here is the detailed trace
```
2015-10-01 14:28:30.378 GetFIT[7293:84533] *** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '-[NSURL isFIFEUrl]: unrecognized selector sent to instance 0x7fb863d24940'
*** First throw call stack:
(
	0   CoreFoundation                      0x0000000103d2ff65 __exceptionPreprocess + 165
	1   libobjc.A.dylib                     0x0000000104f71deb objc_exception_throw + 48
	2   CoreFoundation                      0x0000000103d3858d -[NSObject(NSObject) doesNotRecognizeSelector:] + 205
	3   CoreFoundation                      0x0000000103c85f7a ___forwarding___ + 970
	4   CoreFoundation                      0x0000000103c85b28 _CF_forwarding_prep_0 + 120
	5   GetFIT                              0x0000000103438374 -[GIDProfileData imageURLWithDimension:] + 75
	6   GetFIT                              0x0000000103261caf -[GoogleUser initWithGIDGoogleUser:] + 383
	7   GetFIT                              0x00000001033a93d2 _TTOFCSo10GoogleUsercfMS_FT13GIDGoogleUserGSQCSo13GIDGoogleUser__GSQS__ + 34
	8   GetFIT                              0x00000001033a726f _TFCSo10GoogleUserCfMS_FT13GIDGoogleUserGSQCSo13GIDGoogleUser__GSQS__ + 79
	9   GetFIT                              0x00000001033a57df _TFC6GetFIT9GFLoginVC6signInfS0_FTGSQCSo9GIDSignIn_16didSignInForUserGSQCSo13GIDGoogleUser_9withErrorGSQCSo7NSError__T_ + 479
	10  GetFIT                              0x00000001033a6348 _TToFC6GetFIT9GFLoginVC6signInfS0_FTGSQCSo9GIDSignIn_16didSignInForUserGSQCSo13GIDGoogleUser_9withErrorGSQCSo7NSError__T_ + 88
	11  GetFIT                              0x0000000103440018 __37-[GIDSignIn addCallDelegateCallback:]_block_invoke + 123
	12  GetFIT                              0x0000000103437c16 -[GIDCallbackQueue fire] + 147
	13  GetFIT                              0x000000010343fc2b __38-[GIDSignIn addDecodeIdTokenCallback:]_block_invoke_2 + 385
	14  libdispatch.dylib                   0x00000001077a4ef9 _dispatch_call_block_and_release + 12
	15  libdispatch.dylib                   0x00000001077c549b _dispatch_client_callout + 8
	16  libdispatch.dylib                   0x00000001077ad34b _dispatch_main_queue_callback_4CF + 1738
	17  CoreFoundation                      0x0000000103c903e9 __CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__ + 9
	18  CoreFoundation                      0x0000000103c51939 __CFRunLoopRun + 2073
	19  CoreFoundation                      0x0000000103c50e98 CFRunLoopRunSpecific + 488
	20  GraphicsServices                    0x000000010b981ad2 GSEventRunModal + 161
	21  UIKit                               0x0000000105c85676 UIApplicationMain + 171
	22  GetFIT                              0x00000001032f4f0d main + 109
	23  libdyld.dylib                       0x00000001077f992d start + 1
)
libc++abi.dylib: terminating with uncaught exception of type NSException
```
![screen shot 2015-10-01 at 2 27 50 pm](https://cloud.githubusercontent.com/assets/5868927/10216774/1dea7fda-6849-11e5-9a58-8e0ec8d658c8.png)

**Comment by silvolu** [comment]
@sauvikatinnofied thanks for your feedback, I can reproduce your issue. I'll follow this up internally, and I'll update you as soon as I have news.

**Comment by silvolu** [comment]
@sauvikatinnofied this is fixed in 2.3.1 that we've just released.

**Comment by sauvikatinnofied** [comment]
@silvolu thank you very much for your quick support. I upgraded to the the sdk to latest one (2.3.1). The issue has been resolved. Thanks a lot.

**Comment by jlubeck** [comment]
THANK YOU! I can validate that this was broken on 2.3.0 and now it's fixed on 2.3.1
